sweep/sweeper.go

```diff
@@ -111,6 +111,28 @@ type inputCluster struct {
 	inputs       pendingInputs
 }
 
+// pendingSweepsReq is an internal message we'll use to represent an external
+// caller's intent to retrieve all of the pending inputs the UtxoSweeper is
+// attempting to sweep.
+type pendingSweepsReq struct {
+	respChan chan []PendingSweep
+}
+
+// PendingInput represents an input the UtxoSweeper is currently attempting to
+// sweep.
+type PendingInput struct {
+	OutPoint wire.OutPoint
+	FeeRate  lnwallet.SatPerKWeight
```

Comment:
Should we expose more sweeper state, as this is a pretty technical rpc already? `minPublishHeight` and `publishAttempts`, `witnessType`. Would be useful for investigating problems in the field too. In any case, `value` seems like a useful one to add (`input.SignDesc().Output.Value`).

Reply:
Good idea, done.

Reply:
Maybe a comment should be added saying that those fields that are dependent on the internal sweeper algorithm (like `publishAttempts`) can change with every release. So that we don't need to go through the deprecate cycle if we want to change anything internally.

Reply:
Done.